crawler-cli/src/test/java/org/gbif/crawler/pipelines/dwca/DwCAToAvroPathsTest.java

```diff
@@ -0,0 +1,111 @@
+package org.gbif.crawler.pipelines.dwca;
+
+import org.gbif.api.model.crawler.DwcaValidationReport;
+import org.gbif.api.vocabulary.DatasetType;
+import org.gbif.common.messaging.api.messages.DwcaValidationFinishedMessage;
+
+import java.io.File;
+import java.net.URI;
+import java.util.UUID;
+
+import org.apache.hadoop.fs.Path;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+/**
+ * Test dwca-to-avro commands Configurations with received message parameter verification
+ */
+public class DwCAToAvroPathsTest {
```

Comment:
testInvalidInputDirectory , testInvalidOutputDirectory,  testInvalidDatasetDirectory and testAllPos are almost identical, maybe we should have a private reusable method for all those test cases?